README.md

```diff
@@ -62,6 +63,23 @@ _For mimicking an email inbox (ala mailtrap) for local usage._
   * `MAIL_PASSWORD=null`
   * `MAIL_ENCRYPTION=null`
 
+### MinIO
+_For running an S3 service locally._
+
+* Edit the `/etc/hosts` file on your system and make a reference for `127.0.0.1 s3.docker`
```

Comment:
We could probably retire this note. We don't require the other services to make a loopback pretty url. We've just been reserving localhost ports to use. For consistency I say we remove it, so its just localhost:9090 - then at some point in future if we think so we give clever docker domains to all.

Reply:
Okay, we do need this because of the `minio.hostname` property. I want to research this a bit more before I +1 or -1.

Reply:
I found my notes on why we had to use this solution.  It was to have temporary urls function correctly. 
It was either this or creating a separate config for minio in `config/filesystems.php`

We eventually decided on this solution because the only changes required are in the `.env` file.
see: https://tkacz.pro/laravel-temporary-urls-minio/